features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```diff
@@ -62,7 +106,12 @@ repos:
   cpp:
     - name: cuml
       sub_dir: cpp
-      depends: [rmm, raft, cumlprims_mg]
+      depends: [fmt, rmm, raft, cumlprims_mg]
+      args: "-DSINGLEGPU=OFF -DDETECT_CONDA_ENV=OFF -DBUILD_CUML_C_LIBRARY=OFF -DBUILD_CUML_CPP_LIBRARY=ON -DBUILD_CUML_EXAMPLES=OFF -DBUILD_CUML_TESTS=OFF -DBUILD_PRIMS_TESTS=OFF -DBUILD_CUML_BENCH=OFF -DBUILD_CUML_PRIMS_BENCH=OFF -DCUML_RAFT_CLONE_ON_PIN=OFF -DDISABLE_DEPRECATION_WARNINGS=ON"
```

Comment:
I tried doing `args: |`, but I think it had trouble with that. I should try again.

Reply:
yeah, it ends up interpolating to:
```bash
cmake_args+=(-DSINGLEGPU=OFF;local -DDETECT_CONDA_ENV=OFF;local -DBUILD_CUML_C_LIBRARY=OFF;local -DBUILD_CUML_CPP_LIBRARY=ON;local -DBUILD_CUML_EXAMPLES=OFF;local -DBUILD_CUML_TESTS=OFF;local -DBUILD_PRIMS_TESTS=OFF;local -DBUILD_CUML_BENCH=OFF;local -DBUILD_CUML_PRIMS_BENCH=OFF;local -DCUML_RAFT_CLONE_ON_PIN=OFF;local -DDISABLE_DEPRECATION_WARNINGS=ON;local );
```

Reply:
fixed.